TaskNo29/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿internal class Program
+{
+    private static void Main(string[] args)
+    {
+        Console.OutputEncoding = System.Text.Encoding.Unicode;
+
+        Console.WriteLine();
+        Console.WriteLine("Part No. 1");
+        Console.WriteLine();
+
+        FormatuotasTEkstas();
+    }
+
+    private static void FormatuotasTEkstas()
```

Comment:
Pagal užduotį duomenų įvedimas turėtų vykti `Main` metode ir paskui šis metodas turėti tik išvesti informaciją kuri jam turėtų būti perduota.